# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: accessible/android/moz.build
# Commit: c38e72058869
# Full Hash: c38e720588697e5c0917cab88a4fea6d70af71e1
# Author: Eitan Isaacson <eitan@monotonous.org>
# Date: 2019-09-24 03:42:17
# Regressor Bug: 1564549
# File Overlap Count: 1
# Description:
#   Bug 1564549 - Introduce native traveral rule. r=yzen
#   
#   The rule takes a granularity constant as an argument in the constructor.
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D45598
# ==============================================================================

diff -r aca91e9add16 -r c38e72058869 accessible/android/moz.build
--- a/accessible/android/moz.build	Mon Sep 23 21:06:40 2019 +0000
+++ b/accessible/android/moz.build	Mon Sep 23 21:06:49 2019 +0000
@@ -7,6 +7,7 @@
 EXPORTS.mozilla.a11y += ['AccessibleWrap.h',
     'HyperTextAccessibleWrap.h',
     'SessionAccessibility.h',
+    'TraversalRule.h',
 ]
 
 SOURCES += [
@@ -16,6 +17,7 @@
     'ProxyAccessibleWrap.cpp',
     'RootAccessibleWrap.cpp',
     'SessionAccessibility.cpp',
+    'TraversalRule.cpp',
 ]
 
 LOCAL_INCLUDES += [
@@ -24,6 +26,7 @@
     '/accessible/html',
     '/accessible/ipc',
     '/accessible/ipc/other',
+    '/accessible/xpcom',
     '/accessible/xul',
     '/dom/base',
     '/widget',